iree/testing/vulkan/vulkan_gui_util.cc

```diff
@@ -35,11 +35,13 @@ std::vector<const char*> GetIreeLayers(
     iree_hal_vulkan_extensibility_set_t extensibility_set,
     iree_hal_vulkan_features_t features) {
   iree_host_size_t required_count;
-  iree_hal_vulkan_get_layers(extensibility_set, features, 0, NULL,
-                             &required_count);
+  iree_hal_vulkan_query_extensibility_set(
+      features, extensibility_set, /*string_capacity=*/0,
```

Comment:
Swapping the argument ordering didn't produce a compiler here error since 

```
typedef uint64_t iree_hal_vulkan_features_t;
...
typedef uint32_t iree_hal_vulkan_extensibility_set_t;
```

:(